src/lib/errors/index.js

```diff
@@ -39,6 +39,7 @@ const MojaloopApiErrorCodes = {
     UNACCEPTABLE_VERSION:             { code: '3001', message: 'Unacceptable version requested', httpStatusCode: 406 },
     UNKNOWN_URI:                      { code: '3002', message: 'Unknown URI', httpStatusCode: 404 },
     ADD_PARTY_INFO_ERROR:             { code: '3003', message: 'Add Party information error' },
+    DELETE_PARTY_INFO_ERROR:          { code: '3004', message: 'Delete Party information error' },
```

Comment:
If this is a custom error code (please let me know the context here), it has be 3040 or higher. 3004 is reserved for future use by the API.

Please refer to section 7.6 of the API Definition v1.0 (page 147). Below is the section from the spec
```
All specific errors below xx40; that is, xx00 to xx39, are reserved for future use by the API. All specific errors above and including xx40 can be used for scheme-specific errors. If a client receives an unknown scheme-specific error, the unknown scheme-specific error should be interpreted as a generic error for the high- and low-level category combination instead (xx00).
```

Reply:
@Sam Please see here https://github.com/mojaloop/account-lookup-service/pull/145/files#diff-3996b483a30897f8110decbf594713b3R393-R426 for context. Would change the code.